src/Tools/onnx-mlir-opt/onnx-mlir-opt.cpp

```diff
@@ -29,17 +29,14 @@
 #include <mlir/Support/FileUtilities.h>
 #include <mlir/Support/MlirOptMain.h>
 
+#include "src/Accelerators/NNPA/Dialect/ZHigh/ZHighOps.hpp"
+#include "src/Accelerators/NNPA/Dialect/ZLow/ZLowOps.hpp"
 #include "src/Dialect/Krnl/KrnlOps.hpp"
 #include "src/Dialect/ONNX/ONNXOps.hpp"
 #include "src/InitMLIRPasses.hpp"
 #include "src/InitOMPasses.hpp"
 #include "src/Pass/Passes.hpp"
 
-#ifdef __NNPA__
```

Comment:
I placed the #ifdef __NNPA__ inside the NNP specific headers rather than here.

Reply:
prefer the old style

Reply:
reverted to old style